packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -438,10 +440,14 @@ export const TracksTable = ({
             includeAddToAlbum: false
           }
         : {
+            includeShare: !isUnlisted || isOwner,
+            includeEmbed: !isUnlisted,
             includeEdit: !disabledTrackEdit,
             includeAddToPlaylist:
-              !isStreamGated ||
-              (isContentUSDCPurchaseGated(streamConditions) && hasStreamAccess),
+              !isUnlisted &&
+              (!isStreamGated ||
+                (isContentUSDCPurchaseGated(streamConditions) &&
+                  hasStreamAccess)),
```

Comment:
do we not want to just check access instead? if I have a follow-gated unlock can I not add it to my playlist?

Reply:
not sure where we ended up on that.. we might even want to now allow it regardless of type.

Reply:
gonna merge for now and follow-up if needed